### PR TITLE
[YUNIKORN-1059] Add Chinese translation for get started documents

### PR DESCRIPTION
**Pull request by cdmikechen**
Add Chinese translation for get started documents. This PR contains two contents:
1. Add `core_features.md` and `get_started.md`. Because  the "quick start" chapter in home page is linked to 0.12.1, so 0.12.1 is also added. After the commit comparison, these two files in 0.12.1 is no different from the master. At the same time, several hyperlinks that depend on folders (links to the original file) are added to avoid compile error.
2. Fix a problem that some links in home page redirect to en.

### CODE REVIEW

**Comment by yangwwei** [comment]
hi @yuchaoran2011 can you be the main reviewer for this PR? I'll do some cross-check as well. Thanks

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+YuniKorn的主要特点包括：
+
+## 应用感知调度
+
+YuniKorn 的关键特性之一就是支持应用感知。在默认的K8s调度程序中，它只能根据 Pod 进行调度，而不能基于用户、作业或者队列进行更细粒度的调度。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:31]:
```suggestion
YuniKorn的关键特性之一就是支持应用感知。在默认的K8s调度程序中，它只能根据Pod进行调度，而不能基于用户、作业或者队列进行更细粒度的调度。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+与之不同的是，YuniKorn 可以识别用户、作业或者队列，并在做出调度决策时，考虑更多与它们相关的因素，如资源、排序等。
+这使我们能够对资源配额、资源公平性和优先级进行细粒度控制，这是多租户计算系统最重要的需求。
+
+## 层次资源队列
+
+层次队列提供了一种有效的机制来管理集群资源。
+队列的层次结构可以在逻辑上映射到组织结构。这为不同租户提供了对资源的细粒度控制。
+YuniKorn UI 提供了一个集中的视图来监视资源队列的使用情况，它可以帮助您了解不同租户是如何使用资源的。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:39]:
```suggestion
YuniKorn的用户界面提供了一个集中的视图来监视资源队列的使用情况，它可以帮助您了解不同租户是如何使用资源的。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+此外，用户可以利用设置最小/最大队列容量来为每个租户设定其弹性资源配额。
+
+## 作业排序和排队
+
+YuniKorn 将每个资源队列中的队列进行排队，排序策略决定哪个应用程序可以首先获得资源。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:44]:
```suggestion
YuniKorn将每个资源队列中的作业进行排队。排序策略决定哪个应用程序可以首先获得资源。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+这个策略可以是多种多样的，例如简单的 `FIFO`、`Fair`、`StateAware` 或基于 `Priority` 的策略。
+队列可以维持应用的顺序，调度器根据不同的策略为作业分配相应的资源。这种行为更容易被理解和控制。
+
+此外，当配置队列最大容量时，作业和任务可以在资源队列中正确排队。
+如果剩余的容量不够，它们可以排队等待，直到释放一些资源。这就简化了客户端操作。
+与默认调度程序不同，资源由命名空间资源配额限制，这是由配额许可控制器强制执行的。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:50]:
```suggestion
而在默认调度程序中，资源由命名空间资源配额限制：如果命名空间没有足够的配额，Pod 就不能被创建。这是由配额许可控制器强制执行的。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+如果下面的命名空间没有足够的配额，Pod 就不能被创建。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:51]: can delete this line

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+在多租户环境中，许多用户共享集群资源。
+为了避免租户争夺资源或者可能的资源不足，需要做到更细粒度的公平性需求，以此来实现跨用户以及跨团队/组织的公平性。
+考虑到权重或优先级，一些更重要的应用可以获得超过其配额的更多的需求资源。
+这往往与资源预算有关，更细粒度的公平模式可以进一步提高资源控制。
+
+## 资源预留
+
+YuniKorn 会自动为未完成的请求进行资源预留。
+如果 Pod 无法分配，YuniKorn 将尝试在一个满足条件的节点上保留它，并在这个保留节点上暂时分配 pod（在尝试其他节点之前）。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:64]:
```suggestion
如果 Pod 无法分配，YuniKorn 将尝试把它预留在一个满足条件的节点上，并在这个节点上暂时分配该 pod（在尝试其他节点之前）。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+under the License.
+-->
+
+在阅读本指南之前，我们假设您有一个Kubernetes集群或本地 Kubernetes 开发环境，例如 MiniKube。
+还假定 `kubectl` 在您的环境路径内，并且配置正确。
+遵循此 [指南](../developer_guide/env_setup.md) 来讲述如何使用 docker-desktop 设置本地Kubernetes集群。
+
+## 安装
+
+最简单的方法是使用我们的 Helm Charts 在现有的Kubernetes集群上部署YuniKorn。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md:32]:
```suggestion
最简单的方法是使用我们的 Helm Chart 在现有的Kubernetes集群上部署YuniKorn。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+我们建议使用 Helm 3 或更高版本。
+
+```shell script
+helm repo add yunikorn  https://apache.github.io/incubator-yunikorn-release
+helm repo update
+kubectl create namespace yunikorn
+helm install yunikorn yunikorn/yunikorn --namespace yunikorn
+```
+
+默认情况下，helm chart将在集群中安装调度器、web服务器和 admission-controller。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md:42]:
```suggestion
默认情况下，Helm Chart将在集群中安装调度器、web服务器和 admission-controller。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+`admission-controller` 一旦安装，它将把所有集群流量路由到 YuniKorn。
+这意味着资源调度会委托给 YuniKorn。在helm安装过程中，可以通过将 `embedAdmissionController` 标志设置为false来禁用它。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md:44]:
```suggestion
这意味着资源调度会委托给 YuniKorn。在Helm安装过程中，可以通过将 `embedAdmissionController` 标志设置为false来禁用它。
```

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+如果你不想使用 helm charts，您可以找到我们的细节教程 [点击这里](../developer_guide/deployment.md) 。
```
Comment (yuchaoran2011) [inline i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/get_started.md:46]:
```suggestion
如果你不想使用 Helm Chart，您可以找到我们的细节教程 [点击这里](../developer_guide/deployment.md) 。
```

**Review by yuchaoran2011 — CHANGES_REQUESTED** [review]
@cdmikechen Overall great work. I suggested some changes to make thing more clear. Note that I didn't repeat the comments for the 0.12.1 version of the doc, please fix those accordingly as well. A minor cosmetic suggestion is that I noticed you've left spaces around English words, which can be removed.

**Comment by cdmikechen** [comment]
@yuchaoran2011 
Hi~ For the reason of leaving blank spaces in some english words, I found a problem when translating other articles:
If a phrase is `XX YY ZZ`,  it may be better translated into `中文 XXX YY ZZ 中文` than `中文XXX YY ZZ中文` in sensory sense. Therefore, when I encounter english words or phrases that no need to be translated, I will add a space between Chinese and English words.
But after your suggestion, I now think it may be better not to add spaces if it is a word. But if it is a phrase, it may be better to add spaces.

File: i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md (hunk 2 above)
Comment (cdmikechen) [reply i18n/zh-cn/docusaurus-plugin-content-docs/current/get_started/core_features.md:39]: @yuchaoran2011 
`UI` should be a word that can be understood without translation. The original text is `UI`, so I didn't translate it.

**Review by yuchaoran2011 — APPROVED** [review]

**Pull request merged**